libs/core/src/components/grid/grid.stories.mdx

```diff
@@ -0,0 +1,111 @@
+import { Meta, Canvas } from '@storybook/addon-docs'
+import { GdsGrid } from '../../../../../dist/libs/core/src'
+import "./grid.stories.css"
+
+<Meta title="Components/Grid" />
+
+# Grid
+
+The `gds-grid` is a custom element that provides a flexible grid system. It uses CSS grid layout to arrange its child elements into columns. This component is highly customizable and responsive, allowing you to specify the number of `columns`, `gap`, `padding`, and fluidity for different devices with automated column size based on the content using the `fluid` attribute.
+
+## Usage
+
+To use the `gds-grid`, you add it to your HTML and specify the number of columns for desktop, tablet, and mobile devices using the `columns` attribute. The `columns`, `gap` and `padding` attribute takes a string of three tokens separated by spaces. The first number is for `desktop`, the second for `tablet`, and the third for `mobile`.
+
+```html
+<gds-grid columns="8 4 2">
+    <!-- Child elements here -->
+</gds-grid>
+```
+In the example above, the grid will have `8` columns on `desktop`, `4` on `tablet`, and `2` on `mobile`.
+
+## Columns
+
+The `columns` attribute is used to define the number of columns for different devices.
+
+The format is `columns="[DESKTOP] [TABLET] [MOBILE]"`.
```

Comment:
In the upcoming version, I intended to implement a more comprehensive method for managing breakpoints. However, after considering the matter further, I believe that placing the breakpoint beforehand contributes to clarity will go with it ✌️

I had thought of an additional attribute like breakpoints, allowing for the specification of widths like ﻿breakpoints="1200px 600px 400px" but I guess this won't be necessary since it doesn't really change that often, if it is requested on the future would be easy to extend. 

Will update the attribute accordingly 👌

Reply:
Me and Jocke talked about this aswell! It will be messy how to handle/generate the mediaqueries for this, you end up recreating a middle layer for CSS when it's probably just easier to just use CSS in your app to do this configuration. And if you have special requirement for overriding the default stying of this grid component, you are probably better off writing the styling yourself anyway :)

Reply:
This is now updated